### ISSUE TITLE
Add a completion status called "Not beatable"

### ISSUE THREAD

**Issue by unrealed** [JosefNemec/Playnite#2693]
There are a lot of games that are kind of meant to be played forever and cannot be "beaten", whether they're online multiplayer games like League of Legends/Rainbow 6 Siege, or that are designed for replayability like a Spelunky/Slay the Spire.

It would be nice if **there was a completion category called "Not beatable"** that we could set for games. The point of it would be to have the stats tab give us a %age of how many of our games we've beaten as a %age of our total **beatable** games (the total of our games minus the ones marked "Not beatable")

It's should be a pretty simple addition and it would make clearing the backlog easier and more motivating

**Comment by XenorPLxx** [comment]
Completion statuses can be managed by used in Menu > Library > Library Management.

**Comment by unrealed** [comment]
Oh wow! I'd somehow missed that! While it's great I can make that status myself I'd still want it be one of the defaults and I **especially** would like to have that feature of the Stats page showing %age of completion based on beaten games/(total games - not beatable games). 

But that's the actual request now

**Comment by JosefNemec** [comment]
We already show completion statuses on stats view and I don't think we need to add "Not beatable" to default list since people can add it themselves.

**Comment by unrealed** [comment]
> We already show completion statuses on stats view.

Yes, but currently what it does is show you a %age of each completion status based on the **total** number of games, so currently 80% of my **total** 2036 games (I know) are marked Not Played. 

But that doesn't tell me how close I am to beating every game I own that **IS** beatable, which is the stat I want to know.

So what I WISH would happen is that "Not Beatable" was a new completion status added by default so that the Stats page could show that new stat. It could be called "Percentage of beatable games beaten" or something. 

It could be listed under the Stats -> Global -> Completion Status but a few lines belowe the list of each of the individual Completion Statuses (just like Total play time is listed separately in the Overview box)

**Comment by unrealed** [comment]
I guess that stat is the real feature request so let's rename again
